www/nghttp2/Portfile

```diff
@@ -52,49 +44,78 @@ configure.args      --disable-silent-rules \
+    configure.env-append \
+                        "JANSSON_LIBS=-L${prefix}/lib -ljansson" \
+                        "LIBEVENT_OPENSSL_LIBS=-L${prefix}/lib -levent -levent_openssl" \
+                        "OPENSSL_LIBS=-L${prefix}/lib -lcrypto -lssl"
+
+    if {[variant_isset python27]} { set PythonVersion   2.7 }
+    if {[variant_isset python36]} { set PythonVersion   3.6 }
+    if {[variant_isset python37]} { set PythonVersion   3.7 }
+    if {[variant_isset python38]} { set PythonVersion   3.8 }
+
+    if {[variant_isset python27] || [variant_isset python36] || [variant_isset python37] || [variant_isset python38]} {
+        set PythonBranch    [join [lrange [split ${PythonVersion} .] 0 1] ""]
+
+        depends_lib-append \
+                        port:py${PythonBranch}-cython \
+                        port:py${PythonBranch}-setuptools \
+                        port:python${PythonBranch}
+
+        configure.env-append \
+                        CYTHON=${prefix}/bin/cython-${PythonVersion} \
+                        PYTHON=${prefix}/bin/python${PythonVersion}
+
+        destroot.env    PYTHONPATH=${destroot}${prefix}/lib/python${PythonVersion}/site-packages/
+
+        pre-destroot {
+            xinstall -d ${destroot}${prefix}/lib/python${PythonVersion}/site-packages/
+        }
+    } else {
+        configure.args-append \
+                        --disable-python-bindings \
+                        --without-cython
+    }
 
-    pre-destroot {
-        xinstall -d ${destroot}${prefix}/lib/python${PythonVersion}/site-packages/
+    variant python27 conflicts python36 python37 python38 description {Build using Python 2.7} {
+        configure.env-append   "PYTHON_EXTRA_LDFLAGS=-u _PyMac_Error ${frameworks_dir}/Python.framework/Versions/${PythonVersion}/Python"
     }
-} else {
-    configure.args-append \
-                    --disable-python-bindings \
-                    --without-cython
-}
 
-variant python27 conflicts python36 python37 python38 description {Build using Python 2.7} {
-    configure.env-append   "PYTHON_EXTRA_LDFLAGS=-u _PyMac_Error ${frameworks_dir}/Python.framework/Versions/${PythonVersion}/Python"
-}
+    variant python36 conflicts python27 python37 python38 description {Build using Python 3.6} {}
 
-variant python36 conflicts python27 python37 python38 description {Build using Python 3.6} {}
+    variant python37 conflicts python27 python36 python38 description {Build using Python 3.7} {}
 
-variant python37 conflicts python27 python36 python38 description {Build using Python 3.7} {}
+    variant python38 conflicts python27 python36 python37 description {Build using Python 3.8} {}
 
-variant python38 conflicts python27 python36 python37 description {Build using Python 3.8} {}
+    post-destroot {
+        # remove files contained in the libnghttp2 subport
+        file delete -force ${destroot}${prefix}/lib
+        file delete -force ${destroot}${prefix}/include
```

Comment:
Note that `delete` is the same as `file delete -force`.

Reply:
Fixed.